cookienotice/cookienotice.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/**
+ * Name: Cookie Notice
+ * Description: Configure, show and handle a simple cookie notice
+ * Version: 1.0
+ * Author: Peter Liebetrau <https://socivitas/profile/peerteer>
+ * 
+ */
+use Friendica\Core\Hook;
+use Friendica\Core\Config;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+
+/**
+ * cookienotice_install
+ * registers hooks
+ * 
+ * @return void
+ */
+function cookienotice_install()
+{
+	$file = 'addon/cookienotice/cookienotice.php';
```

Comment:
Tip: This string is actually contained in the magic constant `__FILE__`. If you ever rename your file for whatever reason, this constant will update accordingly.